src/tests/ui/planner/toolbarHeader.ts

```diff
@@ -70,4 +71,8 @@ export class ToolbarHeader extends BaseElement {
     await this.titleTextInput.enterText(title);
     await this.saveFilterBtn.clickWhenReady();
   }
+
+  async getFilterConditions() {
+    return this.activeFiltersList;
```

Comment:
You need to do `getTextWhenReady()` on `activeFiltersList`. This returns the page object representing the list and not it's actual value.